.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,98 @@
+name: Bug report
+description: Something isn't working as expected
+#title: ""
+labels: 
+  - Bug
+#assignees: ""
+body:
+  - type: markdown
+    id: overall
+    attributes:
+      value: |
+        Please follow this Issue template to provide relevant information, such as source code repository, website URL and screenshots, which will help us investigate.
+        请按照此 Issue 模版提供相关信息，例如源码仓库、网站链接和屏幕截图，这将有助于我们进行调查。
+  - type: checkboxes
+    id: checklist
+    attributes:
+      label: Issue Checklist
+      description: |
+        (我确认我已经查看了)
+        Change [ ] to [x] to select (将 [ ] 换成 [x] 来选择)

```

Comment:
This line should be able to be deleted, since the user can directly click the checkbox instead of modifying the markdown

Reply:
Resolved.